terraform/ec2/linux/variables.tf

```diff
@@ -8,7 +8,7 @@ variable "region" {
 
 variable "ec2_instance_type" {
   type    = string
-  default = "t3a.medium"
+  default = "t3a.large"
```

Comment:
Why is the default instance need to be t3a.large?

Reply:
My bad. I intend to increase it for Windows, not for Linux. However, fixing the OOM issue with Windows and decrease it to `t3a.medium`